DistributionUpdate/PackageUpdate/PackageInfoTools.g

```diff
@@ -336,23 +336,34 @@ UpdatePackageInfoFiles := function(pkgdir)
                 "\n  This is not allowed, so the info file will not be changed\n");
           has_error := true;
         fi;
-        if IsBound(infon.Date) then
-          date := infon.Date;
-          if not IsString(date) and Length(date) = 10 and date{[3,6]} = "//" and
-            ForAll( date{ [1,2,4,5,7,8,9,10] }, IsDigitChar ) then
-            Print("  ERROR (", info.PackageName, "): the date ", date,
-                  "should be a string of the form `dd/mm/yyyy'\n" );
-            has_error := true;
-          else
-            date := List( SplitString( date, "/" ), Int);
-            if not date[1] in [1..31] and
-                   date[2] in [1..12] and
-                   date[3] > 1999 and # GAP 4 appeared in 1999
-                   date[1] > DaysInMonth( date[2], date[3] ) then
+        if IsBound(info.Date) then
```

Comment:
The old code used this:
```suggestion
        if IsBound(infon.Date) then
```
Which is right? What's the difference between `info` and `infon`? Perhaps we should add some comments to this code *sigh*

Reply:
(just to clarify: the "sigh" did not refer to this PR, but rather towards the fact that the existing code is mostly undocumented)

Reply:
as far as I understand `infon` means new info (thus the n). The new info gets checked and later replaces the old info stored in `info`.